dev-docs/modules/previousAuctionInfo.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page_v2
+page_type: module
+title: Module - Previous Auction Info
+description: Allows publishers and bidders to collect and inject past auction data into future bid requests
+module_code : previousAuctionInfo
+display_name : Previous Auction Info
+enable_download : true
+sidebarType : 1
+
+---
+
+# Previous Auction Info
+
+## Overview
+
+The Previous Auction Info module enables functionality to collect prior auction data for participating bidders and publishers (This feature is `opt-in`, by default it is disabled).
+
+## How to Enable the Module
+
+A Publisher must do the following in order for the module to work:
+
+* Include the Previous Auction Info module within your Prebid.js build: [https://docs.prebid.org/download.html](https://docs.prebid.org/download.html)
+
+* Configure Prebid.js to enable the Previous Auction Info module:
+
+```javascript
+pbjs.setConfig({previousAuctionInfo: { enabled: true, bidders: ['bidderCode1', 'bidderCode2'], maxQueueLength: 10 }})
+```
+
+* Only valid bid requests submitted by bidders who have enabled the Previous Auction Info module will be permitted.
+
+If the requirements above are met, the flow for how the module works is as follows:
+
+1. A Prebid.js auction runs and completes.
+1. At the end of an auction, details about the auction are collected from each bidder using the module.
+1. If a Prebid bid wins, then the `rendered` field is updated to `1` to indicate this in the collected auction data for all bidders who bid on the same adunit within the same Prebid auction.
+1. During the next Prebid.js auction, if a bidder has this module enabled AND submits a valid bid request, then previous auction info data will be injected into the bidder's bid request of the new auction within the following path: `ortb2.ext.prebid.previousauctioninfo`.
```

Comment:
if a bidder has this module enabled -> if a bidder is included in `bidders` (bidders no longer enable themselves)
submits a valid bid request -> is included in the auction (bidders do not submit bid requests, unless you're referring to those that go to their SSPs)